src/display/text_layer.js

```diff
@@ -56,7 +56,12 @@ var renderTextLayer = (function renderTextLayerClosure() {
     return !NonWhitespaceRegexp.test(str);
   }
 
-  function appendText(textDivs, viewport, geom, styles) {
+  function appendText(textDivs,
```

Comment:
Convention for the PDF.js project is to not use a new line for every arguments as this list is taking up quite some space. We keep the arguments on one line and only add a second line if it does not fit into 80 characters anymore. In this case it would be preferred to have:

<pre>
function appendText(textDivs, viewport, geom, styles, bounds,
                                   enhanceTextSelection) {
  ...
}
</pre>